packages/expo-module-template-local/android/src/main/java/{%= project.package %}/{%- project.moduleName %}.kt

```diff
@@ -39,9 +39,11 @@ class <%- project.moduleName %> : Module() {
     // the view definition: Prop, Events.
     View(<%- project.viewName %>::class) {
       // Defines a setter for the `name` prop.
-      Prop("name") { view: <%- project.viewName %>, prop: String ->
-        println(prop)
+      Prop("url") { view: <%- project.viewName %>, url: String ->
+        view.webView.loadUrl(url.toString())
```

Comment:
The `url` is a string. Why did you call `toString` on it? Should this prop instead be of type `Url`?

Reply:
Seems I missed porting over a change from a compiled template. Thanks!